pass-data-client/src/main/java/org/dataconservancy/pass/client/PassClientDefault.java

```diff
@@ -57,6 +57,11 @@ public URI createResource(PassEntity modelObj) {
         return crudClient.createResource(modelObj);
     }
 
+    @Override
+    public <T extends PassEntity> T createAndReadResource(PassEntity modelObj, Class<T> modelClass) {
+        return crudClient.readResource(crudClient.createResource(modelObj), modelClass);
```

Comment:
It would be interesting to check if `Prefer: return=representation` has been implemented.  I don't think so, but it'd eliminate a roundtrip if so
https://jira.duraspace.org/browse/FCREPO-1879

Reply:
Cool will look at that!